benchmarking/nursery/benchmark_multiobjective/baselines.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Dict, Any, Optional, List, Union
+
+from benchmarking.commons.baselines import MethodArguments
+from benchmarking.commons.default_baselines import (
+    RandomSearch,
+    BayesianOptimization, _baseline_kwargs,
+)
+from syne_tune.config_space import Float, Integer, Categorical
+from syne_tune.optimizer.baselines import MOREA, _assert_searcher_must_be
+from syne_tune.optimizer.schedulers import FIFOScheduler
+from syne_tune.optimizer.schedulers.multiobjective.linear_scalarizer import \
+    LinearScalarizedScheduler
+from syne_tune.optimizer.schedulers.searchers import StochasticSearcher, StochasticAndFilterDuplicatesSearcher
+
+NUM_TO_SAMPLE = 1000
+
+
+def MOREABench(method_arguments: MethodArguments, **kwargs):
+    kwargs = _baseline_kwargs(method_arguments, kwargs)
+    for name, space in kwargs["config_space"].items():
+        if isinstance(space, Float) or isinstance(space, Integer):
+            space = Categorical(categories=space.sample(size=NUM_TO_SAMPLE))
```

Comment:
Why are you doing this? Can the method not deal with types other than `choice`?

If you run this config space into something involving BO, you'll get an enormous input vector, because of 1-hot encoding. It is much better to use the `Ordinal` type, where you need to sort the numbers first.

Reply:
Another problem is that for `Integer`, you may sample duplicates.

Reply:
I'd first check why `MOREA` does not support all types we have. This should not be a problem at all.

Reply:
I checked: `MOREA` is in `optimizer/baselines.py`, and it supports all types, so there is no need to change the config space at all.

Reply:
This is a problem with MOREA, it crashes when `Float` is used. This is a hack around it but indeed should be merged inside MOREA directly, I will look into this.

Reply:
I'd honestly prefer if this gets fixed instead. When looking at the code, there is not reason for it not working with `Float`.

Reply:
@jgolebiowski created https://github.com/awslabs/syne-tune/issues/671 to track the issue